crates/uv/src/requirements.rs

```diff
@@ -234,6 +236,17 @@ impl RequirementsSpecification {
                     })?);
                 }
 
+                if requirements.is_empty() {
+                    if pyproject_toml.build_system.is_some_and(|build_system| {
+                        build_system
+                            .requires
+                            .iter()
+                            .any(|v| v.name.as_dist_info_name().starts_with("poetry"))
+                    }) {
+                        warn_user!("`{}` does not contain any dependencies (hint: Poetry's format is not supported for now)", path.normalized_display());
```

Comment:
Maybe this should be more specific, e.g. `hint: specify dependencies in project.dependencies, tool.poetry.dependencies is not currently supported`. I can imagine users taking "Poetry's format" to mean all kinds of things, like literally the use of pyproject.toml

Reply:
Nice idea: https://github.com/astral-sh/uv/pull/1730